types/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@symfony-types/stimulus-bridge",
+  "description": "Types for the Symfony Stimulus integration bridge",
+  "version": "1.1.0",
```

Comment:
Is this version supposed to be sync with the bridge? Not sure it's useful and it's going to be forgotten for sure :) . I suggest we stick with 1.0